modules/rostests/apitests/advpack/DelNode.c

```diff
@@ -425,7 +425,9 @@ static void Test_DelNodeA(void)
                 }
                 else
                 {
-                    fclose(fopen(path, "w"));
+                    FILE *fout = fopen(path, "w");
+                    if (fout)
+                        fclose(fout);
```

Comment:
What @JoachimHenze asks basically (seeing the current change), is whether basically, `fclose(NULL);` crashes on Windows or not -- _**depending on whether a release or a debug version of the CRT is being used**_ .

Reply:
![fclose-NULL](https://github.com/reactos/reactos/assets/2107452/864cb4f6-a927-44e3-99c1-b682420065b9)
Doesn't crash.

![reactos](https://github.com/reactos/reactos/assets/2107452/7db3aded-e14f-468c-a212-fe7c971b6fe4)
ReactOS also doesn't crash.

Reply:
This phenomenon was conditional.